### PR TITLE
Order of predictions in RNNLearner.get_preds()

### PR DESCRIPTION
**Pull request by StatisticDean**
Referencing this issue : https://github.com/fastai/fastai/issues/975
This PR makes RNNLearner input predictions in the original order. This PR also contains a test that checks  if the true value contained in get_preds[1] is equal to the true_value that was specified when creating the TextClasDataBunch.

I also hesitated to add a test that verified that get_preds(ordered=False) and get_preds(ordered=True) had the same loss. The reason for that is that the output might go through a sigmoid/softmax to be probabilities and you can't just take the loss from the learner. I might add a test later, but feel free to do it yourself.

### CODE REVIEW

**Comment by sgugger** [comment]
Thanks for the fix and the test. I just think one little thing can be refactored (left a comment), if you could make that modification and test it doesn't break anything, that'd be great.

File: fastai/text/learner.py [hunk 1]
```diff
@@ -66,6 +67,19 @@ def load_pretrained(self, wgts_fname:str, itos_fname:str):
         wgts = convert_weights(wgts, old_stoi, self.data.train_ds.vocab.itos)
         self.model.load_state_dict(wgts)
 
+    def get_preds(self, ds_type:DatasetType=DatasetType.Valid, with_loss:bool=False, n_batch:Optional[int]=None, pbar:Optional[PBar]=None, ordered=True) -> List[Tensor]:
+        "Return predictions and targets on the valid, train, or test set, depending on `ds_type`."
+        lf = self.loss_func if with_loss else None
+        preds = get_preds(self.model, self.dl(ds_type), cb_handler=CallbackHandler(self.callbacks),
```
[the remark this answers is not in the document]
Comment (StatisticDean) [reply fastai/text/learner.py]: You're right. I've made the change.

**Comment by sgugger** [comment]
Perfect!
